fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/RespConstants.java

```diff
@@ -198,4 +201,16 @@ public class RespConstants {
     public static final String OVRIGT_DELSVAR_ID_25 = "25.1";
     public static final String OVRIGT_TEXT_ID = "FRG_25.RBK";
     public static final String OVRIGT_SVAR_JSON_ID_25 = "ovrigt";
+
+    public static final String KONTAKT_CATEGORY_ID = "kontakt";
+    public static final String KONTAKT_CATEGORY_TEXT_ID = "KAT_10.RBK";
+    public static final String KONTAKT_ONSKAS_SVAR_ID_26 = "26";
+    public static final String KONTAKT_ONSKAS_SVAR_JSON_ID_26 = "kontaktMedFk";
+    public static final String KONTAKT_ONSKAS_SVAR_TEXT = "FRG_26.RBK";
```

Comment:
For naming convention consider adding _ID when defining TEXT and DESCRIPTION. For example KONTAKT_ONSKAS_SVAR_TEXT_ID and KONTAKT_ONSKAS_SVAR_DESCRIPTION_ID.

And you notice I wrote DESCRIPTION instead of BESKRIVNING, so please consider that too :-)

Reply:
Added _ID to TEXTs and (now) DESCRIPTIONs.